netflix_prep_install.sh

```diff
@@ -10,7 +10,7 @@ sudo apt-get update 2>&1 | dialog --title "Updating package database and install
 if [ ! -d /usr/share/doc/python-crypto ]; then
 	sudo apt-get install -q python-crypto
 fi
-sudo apt-get install -q -y build-essential python-pip libnss3 libnspr4
+sudo apt-get install -q -y build-essential python-pip libnss3 libnspr4 kodi-inputstream-adaptive
```

Comment:
Not needed for OSMC since inputstream.adaptive is installed by default on OSMC, so i wont merge this